WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionWebViewActivity.java

```diff
@@ -20,12 +22,17 @@
 
     public enum Source {
         STATS("stats"), NOTIFICATIONS("notifications");
+        public static final String KEY = "tracking_source";
         private final String value;
 
         Source(String value) {
             this.value = value;
         }
 
+        public String value() {
```

Comment:
How do you feel about renaming this to `getValue()` for consistency?

Reply:
Sure, Kotlin is affecting me 😄